x-pack/plugins/event_log/scripts/mappings.js

```diff
@@ -18,6 +18,10 @@ exports.EcsKibanaExtensionsMappings = {
           type: 'keyword',
           ignore_above: 1024,
         },
+        action_group_id: {
```

Comment:
cool - these changes to the event log mappings/schema gen tools look good, hope it was straight-forward, please update the relevant docs (READMEs?) if something was unclear

Reply:
Yep! The schema generation was straightfoward.